tests/behat/features/jsonapi.content_rating.feature

```diff
@@ -16,6 +16,13 @@ Feature: JSON API Webform
     And the JSON node "data[0].attributes.elements.was_this_page_helpful" should exist
     And the JSON node "data[0].attributes.elements.comments" should exist
 
+  @api
+  Scenario: Prepend a step to disable the readonly mode from UI
+    Given I am logged in as a user with the "administrator" role
+    When I go to "admin/config/services/jsonapi"
+    Then I select the radio button "Accept all JSON:API create, read, update, and delete operations."
```

Comment:
@vincent-gao Has this configuration been exported or is it only setup for the next test scenario? I'm confused why it would be necessary for testing but not the implementation.

Reply:
Hi @GROwen , this will relate to https://github.com/dpc-sdp/tide_api/pull/51 and https://github.com/dpc-sdp/content-vic-gov-au/pull/627

Reply:
Hi @vincent-gao, I think @GROwen's point is that this step shouldn't be required as `tide_api` should (and it looks like it does) have an update hook that will set this. You shouldn't need to do it in the test.

Reply:
Hey, @GROwen  @anthony-malkoun yes. after https://github.com/dpc-sdp/tide_api/pull/51 merged to tide_api. this issue should be fixed subsequently.